chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -986,3 +986,6 @@ form.formItem.childQuestionNotApplicable=Responding to '%s' is not applicable ba
 attestation.checkInReport.apiResponseNotConsistent=does not match API response
 attestation.checkInReport.assurancesResponseNotConsistent=does not match Assurances response
 attestation.checkInReport.rwtResponseNotConsistent=does not match RWT response
+
+#Redis Errors
+redis.connection.timeout=An error occurred during save.
```

Comment:
We had an example where this happened during listing edit/save, but it seems like it could happen any time we are doing something with the cache. Not necessarily "save". At least, I don't feel certain enough about that to put it in this error message.